unison-cli/src/Unison/CommandLine/Welcome.hs

```diff
@@ -14,7 +14,8 @@ import Prelude hiding (readFile, writeFile)
 
 data Welcome = Welcome
   { onboarding :: Onboarding, -- Onboarding States
-    unisonVersion :: Text
+    unisonVersion :: Text,
+    showWelcomeHint :: Bool
```

Comment:
I'm 50/50 on this being a `Bool` and not a `Text` and in `launch` we just pass in the appropriate hint message?

Reply:
Yeah, we can always change it later if we have some reason to skew off of 50/50.  It could also be a multi-valued thing to choose between instead of the two Bool values.  I guess, at a point, `Text` is the extreme version of multi-valued ;-)